pyproject.toml

```diff
@@ -28,7 +28,9 @@ dependencies = [
     'roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git@main',
     'stcal >=1.3.3',
     'stpipe >=0.4.2',
-    'tweakwcs >=0.8.0'
+    'tweakwcs >=0.8.0',
+    'astroquery >= 0.4.6',
+    'pandas >= 1.5.3'
```

Comment:
After adding `astroquery` to the list of dependencies, I ran into an error (full traceback [here](https://github.com/spacetelescope/romancal/actions/runs/4621406063/jobs/8172984830)) saying that `pandas` was not installed. After going through the traceback, I realized that astropy will raise an error when it doesn't find `pandas` because of the methods `Table.from_pandas()` and `Table.to_pandas()`.

From the `astropy.table.Table.to_pandas` [docs](https://docs.astropy.org/en/stable/api/astropy.table.Table.html#astropy.table.Table.to_pandas) (same thing goes for [`Table.from_pandas`](https://docs.astropy.org/en/stable/api/astropy.table.Table.html#astropy.table.Table.from_pandas)):
```
Raises
------
ImportError
    If pandas is not installed
ValueError
    If the Table has multi-dimensional columns
```